src/api/app/assets/javascripts/webui2/datatables.js

```diff
@@ -5,22 +5,25 @@
 //= require datatables/extensions/FixedColumns/fixedColumns.bootstrap4
 //= require datatables/extensions/FixedColumns/dataTables.fixedColumns
 
+var DEFAULT_DT_PARAMS = {
+  language: { search: '', searchPlaceholder: "Search..." },
+  stateSave: true,
+  stateDuration: 172800 // 2 days
+};
+
 function initializeDataTable(cssSelector, params){ // jshint ignore:line
-  var defaultParams = {
-    language: { search: '', searchPlaceholder: "Search..." },
-  };
-  var newParams = $.extend(defaultParams, params);
+  var newParams = $.extend(DEFAULT_DT_PARAMS, params);
```

Comment:
The first parameter is modified (and returned) in `extend,` which means that `DEFAULT_DT_PARAMS` is modified. I guess that this is only a problem if we have more than one datatable per page.

To avoid problems I would write it:
```Javascript
var newParams = $.extend({}, DEFAULT_DT_PARAMS, params);
```